src/dropDown/dropDown.jsx

```diff
@@ -21,7 +22,7 @@ const Menu = (props) => {
       tabIndex={-1}
       aria-hidden={true}
     >
-      <div className='content'>
+      <div className={`content ${props.dark ? 'dark' : 'light'}`}>
```

Comment:
The dropdown renders strangely for me
![header](https://github.com/agorouniversity/Agoro-UI-Components/assets/47364254/b7ac3e56-9c1b-4948-8d4d-fd462074e36e)

Reply:
Oh the dropdown has padding or some other element at the top there, I think we can leave that for the dropdown rework because it doesn't affect the header.